universal/drm.md

```diff
@@ -1,7 +1,7 @@
 # Fixing DRM support and iGPU performance
 
 * **Note**: Safari 14 and macOS 11, Big Sur are currently unsupported by WhateverGreen's DRM patches. Safari 13 in Catalina and older are supported just fine however.
-* **Note 2**: Browsers not using hardware based DRM(ie. Chrome), will have working DRM without any work both on iGPUs and dGPUs. The below guide is for Safari and other applications with hardware based DRM
```

Comment:
Well, Chromium-based in my opinion, because several others like Opera, Vivaldi, Chromium ungoogled with DRM support, and many others also support 720p DRM content just fine.

Reply:
@vit9696 got it 👍🏻